### DISCUSSION TITLE
Trabalho Eficaz com Código Legado

### DISCUSSION THREAD

**Discussion by lincolixavier** [pilhacheia/clube-do-livro-dev#11] [Books]
Seu código é fácil de alterar? Você recebe feedback quase instantâneo quando o altera? Você o entende? Se a resposta a alguma dessas perguntas for “não”, você tem um código legado e ele está consumindo tempo e dinheiro de seus esforços de desenvolvimento. Neste livro, Michael C. Feathers oferece estratégias completas para um trabalho mais eficaz com bases de código legadas grandes e não testadas.

Preço R$98,85
[Link Amazon](https://www.amazon.com.br/Trabalho-Eficaz-com-C%C3%B3digo-Legado/dp/8582600321)

**Comment by gustavoferrazfontes** [comment]
Eu li esse livro, achei muito bom! Se você já leu Refactoring do Martin Fowler, Clean Code e tem um bom conhecimento sobre principios SOLID e TDD,  perceberá que o livro usa todas essas literaturas como ferramental para mostrar como é possivel evoluir e melhorar um software legado.
O que achei interessante é que "no final do dia", tudo gira em torno de como fazer um bom uso de OOP. O livro é bem prático, cheio de exemplos e de facil entendimento/leitura. Vale a pena ler!
